python-client/giskard/commands/cli_server.py

```diff
@@ -91,6 +92,33 @@ def get_container(version=None, quit_if_not_exists=True) -> Optional[Container]:
             return None
 
 
+def _is_backend_ready(endpoint) -> bool:
+    try:
+        response = requests.get(endpoint)
+        response.raise_for_status()
+        return "UP" == response.json()["status"]
+    except KeyboardInterrupt:
+        raise click.Abort()
+    except BaseException:
```

Comment:
you can add 
```suggestion
    except BaseException:  # noqa NOSONAR
```
to let Sonar know that it's OK